### ISSUE TITLE
Datagrid borders fail contrast ratio for accessibility

### ISSUE THREAD

**Issue by juanjosegutierrez** [vmware-clarity/ng-clarity#354]
**This is a bug report for the `@clr` Angular or UI versions of the design system.
For the web-component implementation of Clarity (`@cds`), visit [vmware-clarity/core](https://github.com/vmware-clarity/core).**

## Describe the bug

In the angular datagrid, the border color of the radio buttons and the border color of the datagrid are too close to the color of the datagrid background.

1. The background color of the datagrid is #ffffff and the border color of the radio button is #b3b3b3. This contrast ratio is 2:1 which is less than the minimum WCAG ratio of 3:1.
2. The background color of the datagrid is #ffffff and the border color of the datagrid border is #cccccc. This contrast ratio is 1.6:1 which is less than the minimum WCAG ratio of 3:1.

## How to reproduce

<img width="785" alt="Screen Shot 2022-10-19 at 12 19 41 PM" src="https://user-images.githubusercontent.com/19143833/196784455-2b0257a3-b802-4550-9e3c-bb6c9db96e14.png">

1.  Check screenshot attached above.

## Expected behavior

The contrast ratio should be above 3:1.

## Versions

**Clarity version:**

- [x] v12.x
- [ ] v13.x
- [ ] Other:

**Framework:**

- [x] Angular
- [ ] React
- [ ] Vue
- [ ] Other:

**Framework version:**
Angular 12

**Device:**

- Type: MacBook
- OS: MacOS
- Browser chrome
- Version 105

## Additional notes

Other borders in the datagrid may have the same issue.

**Comment by ashleyryan** [comment]
What version of clr/angular are you using in v12? In v13, the border of the radio is much darker than your screenshot and meets accessibility guidelines: https://clarity.design/documentation/datagrid/selection-single

<img width="438" alt="image" src="https://user-images.githubusercontent.com/9469374/196802176-8c2a01c2-22ca-4ee9-b0ab-0bad66827296.png">

I would need Accessibility to weigh in on the requirements for borders. Generally our recommendation for any color contrast issues in Clarity is to use the Clarity Core Shim - which brings the colors from Clarity Core to NG and fixes many, many color contrast issues. I'm not sure if it fixes the border color issue though

https://storybook.core.clarity.design/?path=/story/themes-clarity-ui--page
